src/components/Contacts/ContactFlow/ContactFlow.graphql

```diff
@@ -0,0 +1,19 @@
+query HasActiveTask(
+  $accountListId: ID!
+  $tasksFilter: TaskFilterSetInput
+  $after: String
+  $first: Int
+  $sortBy: TaskSortEnum
+) {
+  tasks(
+    accountListId: $accountListId
+    tasksFilter: $tasksFilter
+    after: $after
+    first: $first
+    sortBy: $sortBy
```

Comment:
Some of these variables only have one possible value in the code. Personally, I'd hardcode them instead of making them variables in this operation. Also, `$after` doesn't appear to be used when calling the lazy query and can be removed.

```suggestion
    accountListId: $accountListId
    tasksFilter: $tasksFilter
    first: 1
    sortBy: "START_AT_ASC"
```

Reply:
I have removed the variables which need not be there and hard coded them.